hack/jenkins/job-configs/kubernetes-build.yaml

```diff
@@ -0,0 +1,64 @@
+- job-template:
+    name: 'kubernetes-{build}'
+    description: 'Grab the latest from GitHub, build. Test owner: Build Cop.'
+    logrotate:
+        numToKeep: 200
+    builders:
+        - shell: '{shell}'
+    publishers:
+        - build-publisher:
+            publish-unstable-builds: true
+            public-failed-builds: true
+        - claim-build
+        - email-ext:
+            recipients: $DEFAULT_RECIPIENTS, cloud-kubernetes-team@google.com
+            presend-script: $DEFAULT_PRESEND_SCRIPT
+            fail: true
+            fixed: true
+            send-to:
+                - culprits
+                - recipients
+        - google-cloud-storage:
+            credentials-id: kubernetes-jenkins
+            uploads:
+                - build-log:
+                    log-name: build-log.txt
+                    storage-location: gs://kubernetes-jenkins/logs/$JOB_NAME/$BUILD_NUMBER
+                    share-publicly: true
+                    upload-for-failed-jobs: true
+        - logparser:
+            parse-rules: /jenkins-master-data/log_parser_rules.txt
+            unstable-on-warning: false
+            fail-on-error: false
+    scm:
+        - git:
+            url: https://github.com/kubernetes/kubernetes
+            branches:
+                - '{branch}'
+            browser: githubweb
+            browser-url: https://github.com/kubernetes/kubernetes
+            wipe-workspace: false
+            skip-tag: true
+    triggers:
+        - pollscm:
+            cron: 'H/2 * * * *'
+    wrappers:
+        - timeout:
+            timeout: 30
+            abort: true
+        - timestamps
+
+- project:
+    name: kubernetes-builds
+    build:
+        - 'build':
+            branch: 'master'
+            shell: 'source ./hack/jenkins/upload-to-gcs.sh; ./hack/jenkins/build.sh'
```

Comment:
we can probably remove the upload-to-gcs.sh bit. it won't work if these builds run on anything besides the jenkins master.